packages/tupaia-web/src/layout/AuthModal.tsx

```diff
@@ -4,97 +4,64 @@
 import { Modal } from '../components';
 import styled from 'styled-components';
 import { TUPAIA_LIGHT_LOGO_SRC } from '../constants';
-import { Button, Typography } from '@material-ui/core';
-import { FONT_SIZES } from '../theme';
-import { OutlinedButton } from '@tupaia/ui-components';
+import { Typography } from '@material-ui/core';
+import { Button, OutlinedButton } from '@tupaia/ui-components';
 
 const Logo = styled.img`
   min-width: 110px;
-  margin-top: 1.8em;
-  margin-bottom: 3em;
+  margin-bottom: 3.6rem;
 `;
 
 const Title = styled(Typography)`
-  font-size: ${FONT_SIZES.viewTitle};
+  font-size: 2rem;
   font-weight: 500;
 `;
 
 const Subtitle = styled(Typography)`
-  font-size: 1em;
-  margin-top: 1.4em;
+  font-size: 0.875rem;
+  line-height: 1.3;
+  margin-top: 1rem;
 `;
 
-const PrimaryButton = styled(Button).attrs({
-  variant: 'contained',
-  color: 'primary',
-})`
+export const ModalButton = styled(Button)`
   text-transform: none;
-  font-size: 1em;
+  font-size: 1rem;
   width: 100%;
   margin-left: 0 !important;
-  margin-top: 2em;
+  margin-top: 2rem;
 `;
 
-const SecondaryButton = styled(OutlinedButton).attrs({
+export const ModalCancelButton = styled(OutlinedButton).attrs({
   color: 'default',
 })`
   text-transform: none;
-  font-size: 1em;
+  font-size: 1rem;
   width: 100%;
   margin-left: 0 !important;
-  padding: 0.375em 1em; // to match the height of the primary button
+  padding: 0.375rem 1rem; // to match the height of the primary button
   border-color: ${({ theme }) => theme.palette.text.secondary};
-  ${PrimaryButton} + & {
-    margin-top: 1.3em;
+  ${ModalButton} + & {
+    margin-top: 1.3rem;
   }
 `;
 
-type ButtonProps = {
-  onClick: () => void;
-  text: string;
-};
 interface AuthModalProps {
   children?: ReactNode;
-  onClose?: () => void;
+  onClose: () => void;
   title?: string;
   subtitle?: string;
-  navigateTo?: To | number; // the path to navigate to on close, if set. Numbers are acceptable for e.g. -1 meaning back 1 route
-  primaryButton?: ButtonProps;
-  secondaryButton?: ButtonProps;
 }
 
-export const AuthModal = ({
-  children,
-  onClose,
-  navigateTo = -1,
-  title,
-  subtitle,
-  primaryButton,
-  secondaryButton,
-}: AuthModalProps) => {
-  const navigate = useNavigate();
-  const onCloseModal = () => {
-    // navigate back to the previous route on close if no path is provided
-    navigate(navigateTo as To);
-    if (onClose) onClose();
-  };
-  // This modal will always be open in this case, because these are only visible on certain routes
+export const AuthModal = ({ children, title, subtitle, onClose }: AuthModalProps) => {
   return (
-    <Modal isOpen={true} onClose={onCloseModal}>
+    <Modal isOpen={true} onClose={onClose}>
       <Logo src={TUPAIA_LIGHT_LOGO_SRC} alt="Tupaia Logo" />
       <Title variant="h2">{title}</Title>
       {subtitle && <Subtitle variant="h3">{subtitle}</Subtitle>}
       {children}
-      {primaryButton && (
```

Comment:
There are a few small variations in the auth modals in the designs such as the forgot password link so I refactored this to leave the button placements up the children prop.